src/Compilers/VisualBasic/Portable/Parser/ParseConditional.vb

```diff
@@ -460,6 +472,44 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             Return SyntaxFactory.ReferenceDirectiveTrivia(hashToken, referenceKeyword, file)
         End Function
 
+        Private Function ParseShebangDirective(hashToken As PunctuationSyntax) As ShebangDirectiveTriviaSyntax
+            Dim builder = New StringBuilder()
+            Debug.Assert(CurrentToken.Kind = SyntaxKind.ExclamationToken,
+                         NameOf(ParseShebangDirective) & " called with wrong token")
+            Dim exclamationToken = DirectCast(CurrentToken, PunctuationSyntax)
+            GetNextToken()
+
+            ' Eat tokens until the end of line.
+            While CurrentToken.Kind <> SyntaxKind.EndOfLineTrivia AndAlso
+                    CurrentToken.Kind <> SyntaxKind.EndOfFileToken AndAlso
+                    CurrentToken.Kind <> SyntaxKind.StatementTerminatorToken
+                builder.Append(CurrentToken.ToFullString())
+                GetNextToken()
+            End While
+
+
+            exclamationToken = exclamationToken.AddTrailingTrivia(SyntaxFactory.CommentTrivia(builder.ToString()))
```

Comment:
I understand this was specifically requested by the IDE team, but it seems strange to me that a directive would be colored as trivia.